src/components/site-header/demo/data--ec.js

```diff
@@ -22,6 +22,25 @@ module.exports = {
     label: 'Log out',
     href: exampleLink,
   },
+  custom_action: {
+    label: 'Notifications',
```

Comment:
even if there is an overlay we should have an href defined, otherwise the link is not valid

Reply:
yes, added the href.